cfp-zh.html

```diff
@@ -0,0 +1,43 @@
+---
+title: Workshop overview
```

Comment:
This should have been "Call for Participation" in the English version - could this be translated accordingly?

Reply:
while I'm at it, I'm seeing that the title of the workshop overview page hasn't been translated in Chine either

Reply:
I've just translated the titles of "overview" and "call for participation".